docs/source/relayer.rst

```diff
@@ -178,33 +133,25 @@ Example: connectiong to testnet
     ENS_DOMAIN_2='my-id.test'
     ENS_DOMAIN_3='my-super-domain.test'
 
+.. _ens-registration:
+
 ENS registration
```

Comment:
To use universal login with your own ENS domain, you will need to register it, connect to the resolver and deploy own registrar. There is a script for that. 
Note: script currently only works for `.test` domains. Tested on Rinkeby and Ropsten test networks.
You can register domain form command line interface and programmatically.